File: Exemplos M10/App_IntentDigitalHUB_XamarinAndroid/Elgin_IntentDigitalHUB_XamarinAndroid/IntentServices/Bridge/IniciaOperacaoAdministrativa.cs
namespace Xamarin_Android_Intent_Digital_Hub.IntentServices.Bridge
{
    class IniciaOperacaoAdministrativa : BridgeCommand
    {
        readonly int idTransacao;
        readonly string pdv;
        readonly int operacao;

        public IniciaOperacaoAdministrativa(int idTransacao, string pdv, int operacao) : base("IniciaOperacaoAdministrativa")
        {
            this.idTransacao = idTransacao;
            this.pdv = pdv;
            this.operacao = operacao;
        }

        protected override string FunctionParameters()
        {
            return "\"idTransacao\"" + ":" + idTransacao + "," +
                    "\"pdv\"" + ":" + "\"" + pdv + "\"" + "," +
                    "\"operacao\"" + ":" + operacao;
        }
    }
}

File: Exemplos M10/App_IntentDigitalHUB_Ionic/Elgin_IntentDigitalHUB_Ionic-font/src/intentDigitalHubService/BRIDGE/Commands/IniciaOperacaoAdministrativa.ts
import { BridgeCommand } from "../BridgeCommand";

export class IniciaOperacaoAdministrativa extends BridgeCommand {
  readonly idTransacao: number;
  readonly pdv: string;
  readonly operacao: number;

  constructor(idTransacao: number, pdv: string, operacao: number) {
    super("IniciaOperacaoAdministrativa");
    this.idTransacao = idTransacao;
    this.pdv = pdv;
    this.operacao = operacao;
  }

  functionParameters(): object {
    return {
      idTransacao: this.idTransacao,
      pdv: this.pdv,
      operacao: this.operacao,
    };
  }
}
